app/lib/common/providers/notifiers/sync_notifier.dart

```diff
@@ -0,0 +1,93 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:acter/common/models/sync_state/sync_state.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' as ffi;
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+// ignore_for_file: avoid_print
+class SyncNotifier extends StateNotifier<SyncState> {
+  final ffi.Client client;
```

Comment:
I see most of us use this alias to point towards package path and sometimes don't. It could cause some confusion for new contributors when they see ```ffi.Client``` and ```Client```. Should we have some general agreement here?

Reply:
SyncState is existing once from ffi and once from the internals of flutter, that's why I used an alias here...